scripts/dark-reader/dark-reader.ts

```diff
@@ -0,0 +1,62 @@
+import { walk } from "https://deno.land/std/fs/mod.ts";
+import { ensureFile } from "https://deno.land/std/fs/mod.ts";
+import { dirname, fromFileUrl, join } from "https://deno.land/std/path/mod.ts";
+
+// Get the directory of the currently running script
+const scriptDir = dirname(fromFileUrl(import.meta.url));
+
+// Construct the parentDir to thescript.ts/../../styles
+const parentDir = join(scriptDir, "../../styles");
```

Comment:
```suggestion
import { REPO_ROOT } from "@/deps.ts";

const parentDir = join(REPO_ROOT, "styles");
```
Please rename the `parentDir` variable to `stylesDir` or something as well.